rio_tiler/io/base.py

```diff
@@ -373,6 +382,9 @@ def merged_statistics(  # type: ignore
             Dict[str, rio_tiler.models.BandStatistics]: bands statistics.
 
         """
+        if asset_expression:
+            _AssetExpressionWarning
```

Comment:
```suggestion
            _AssetExpressionWarning()
```
Same in other places.

Reply:
🤦